src/main/resources/client.properties

```diff
@@ -22,10 +22,10 @@
 ### if the credentials are provided as environment variable or
 ### an instance role is used to handle permissions
 ### see: http://docs.aws.amazon.com/AWSSdkDocsJava/latest/DeveloperGuide/java-dg-roles.html
-#simianarmy.client.aws.accountKey = fakeAccount
-#simianarmy.client.aws.secretKey  = fakeSecret
+simianarmy.client.aws.accountKey = accountkey
+simianarmy.client.aws.secretKey  = secretkey
```

Comment:
we should make sure we allow for iam roles / environment variables, it should be automatic so we don't need to provide these.

Reply:
Just tested this. When I comment these lines out it does automatically pick up my environment variables. So we should be good.